charts/bandstand-headless-service/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: bandstand-headless-service
-version: 1.0.0
+version: 2.0.0
```

Comment:
I don't think this is being used yet (can't see in backstage) so could make a minor/patch.